sigs.yaml

```diff
@@ -2259,16 +2272,32 @@ sigs:
     owners:
     - https://raw.githubusercontent.com/kubernetes/dns/master/OWNERS
   - name: multi-network
+    contact:
+      slack: sig-network-multi-network
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/multi-network-api/main/OWNERS
     - https://raw.githubusercontent.com/kubernetes-sigs/multi-network/main/OWNERS
+    leads:
+    - github: mskrocki
+      name: Maciej Skrocki
+      company: Google
   - name: network-policy
     contact:
       slack: sig-network-policy-api
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/kube-network-policies/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes-sigs/network-policy-api/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/api/master/networking/OWNERS
+    leads:
+    - github: astoycos
+      name: Andrew Stoycos
+      company: Red Hat
```

Comment:
nope :slightly_frowning_face:

Reply:
True, @astoycos what shall we put in here?

Reply:
News to me.  :(

Reply:
@astoycos shall I put Citadel?

Reply:
we can revisit this later when @astoycos gets back to us

Reply:
I think is better to wait, it is ok if he can dedicate only X hours per week/month, but if he is not be able to participate at all it is better to reconsider his commitment, adding a lead that is not going to be present because of reasons is not great

Reply:
Supper sorry For the late response here things are finally starting to settle in for me at the new gig and I'm just starting to catch back up.

So for now if we could just leave my "company" empty that would the best.  I think my available time is going to definitely be much smaller now :/  shooting for around 2 hours a week (usually on the weekends)

Ultimately I would like @tssurya to step in and take over my positions, she already has keep everything rolling over the past month while I've been MIA, and has been super actively involved for over a year now.

Reply:
no need to remove anyone that still have the willingness to participate and still can dedicate some time ...

Reply:
ack I will update the list to include myself if everyone else is OK with that and continue to keep @astoycos with empty company for now. Please let me know if anyone has issues with me doing that...

@astoycos +1 to what @aojea said, any small amount of time you put in is valuable!!

Reply:
I think instead of leaving it empty, we can use "Independent" , that is used in other places for people that does not identify with one particular company

Reply:
ah actually I will update the list to have Andrew in it for now with empty company, given I am not in OWNERS file, its probably not appropriate to add myself till we get that fixed first.